src/test/java/com/endava/internship/mocking/service/PaymentServiceTest.java

```diff
@@ -40,27 +47,35 @@ void setUp() {
         paymentRepository = Mockito.mock(InMemPaymentRepository.class);
```

Comment:
You can stick to others declarations convention and use also @Mock for PaymentRepository.
For PaymentService you can use @InjectMocks instead of calling constructor

Reply:
fixed